ballerina/Package.md

```diff
@@ -1,8 +1,12 @@
 ## Package Overview
 
-The `websubhub` package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This package provides APIs for a WebSub Hub service and WebSub Publisher client.
 
-This package provides the capability to easily implement a WebSub compliant **Hub Service** and **Publisher Client**.
+[**WebSub**](https://www.w3.org/TR/websub/) is a common mechanism for communication between publishers of any kind of web content and their subscribers based on HTTP webhooks. Subscription requests are relayed through hubs, which validate and verify the requests. Hubs then distribute new and updated content to subscribers when it becomes available. WebSub was previously known as PubSubHubbub.
+
+[**WebSub Hub**](https://www.w3.org/TR/websub/#hub) is an implementation that handles subscription requests and distributes the content to subscribers when the corresponding topic URL has been updated.
+
+[**WebSub Publisher**](https://www.w3.org/TR/websub/#publisher) is an implementation that advertises a topic and hub URL on one or more resource URLs.
```

Comment:
Shall we add sub-sections as well with code snippets?

Reply:
Fixed with [e9db73c302362b33e70a1a7706ad50620f8cc12a](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/434/commits/e9db73c302362b33e70a1a7706ad50620f8cc12a)